src/dataSources/JSON.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+var Null = require('./Null');
+
+/**
+ * @constructor
+ */
+var JSON = Null.extend('JSON',  {
+
+    initialize: function(data, fields, calculators) {
+        this.setData(data, fields, calculators);
```

Comment:
Maybe we should not be setting data implicitly on instantiation, but only subsequently on setData. We can leave as is for now and revisit this on the "launch sequence" ticket.